Tests/Transfer/HyperwalletTransfer.swift

```diff
@@ -0,0 +1,118 @@
+import Hippolyte
+@testable import HyperwalletSDK
+import XCTest
+
+class HyperwalletTransferTests: XCTestCase {
```

Comment:
Can you please rename the file to be HyperwalletTransferTests as well?

Reply:
Done